### PR TITLE
Invalidate cache after generating new GPG key

### LINKED ISSUE #1693: Wizard fails when there are no PGP keys
### Summary
Running `gopass` with no secret keys in the keystore will fail after successful key creation (with a custom passphrase, not even a prompt to use my own?!?): 
```
anomalroil@TLD20:~$ gpg -K
anomalroil@TLD20:~$ gopass
It seems you are new to gopass. Do you want to run the onboarding wizard? [Y/n/q]: y
[init] Initializing a new password store ...
[init] [init] No useable crypto keys. Generating new key pair
[init] [init] [crypto] Key generation may take up to a few minutes
[init] [init] [crypto] Creating key pair ...
[init] [init] [crypto] WARNING: We are about to generate some GPG keys.
[init] [init] [crypto] However, the GPG program can sometimes lock up, displaying the following:
"We need to generate a lot of random bytes."
If this happens, please see the following tips:
https://github.com/gopasspw/gopass/blob/master/docs/entropy.md
What is your name? [anomalroil]: Testou
What is your email? []: test@test.com
Continue? [Y/n/q]: y
[init] [init] [crypto] -> OK
[init] [init] [crypto] Passphrase: many tingly maternal scenic

Error: failed to run onboarding wizard: failed to create new private key: failed to create a useable key pair
```

And yet it created the keypair:
```
anomalroil@TLD20:~$ gpg -K
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 2u
/home/anomalroil/.gnupg/pubring.kbx
-----------------------------------
sec   rsa2048 2021-01-08 [SCEA]
      2C12748C437F7833CB53454DADE652F8C8984545
uid           [ultimate] Testou <test@test.com>
ssb   rsa2048 2021-01-08 [SEA]
```

And the next invocation of gopass works:
```
anomalroil@TLD20:~$ gopass
It seems you are new to gopass. Do you want to run the onboarding wizard? [Y/n/q]: y
[init] Initializing a new password store ...
[ 0] Local store
[ 1] Create a Team
[ 2] Join an existing Team

Select action [0]: 0
0
[init] [init] [local] Initializing your local store ...
Please select a private key for encrypting secrets:
[0] gpg - 0xADE652F8C8984545 - Testou <test@test.com>
Please enter the number of a key (0-0, [q]uit) [0]: 0
Please enter an email address for password store git config []: test@test.com
[init] [init] [local]  -> OK
[init] [init] [local] Configuring your local store ...
[init] [init] [local] Do you want to add a git remote? [y/N/q]: n
[init] [init] [local]  -> OK
gopass> quit
```

To reproduce, specify a new GNUPGHOME in an empty directory to have no private key:
```
$ mkdir ~/testou
$ GNUPGHOME=~/testou gopass
```

### PR DESCRIPTION
**Pull request by dominikschulz**
Fixes #1693

RELEASE_NOTES=[BUGFIX] Invalidate GPG key list after generation

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

File: internal/action/init.go [hunk 1]
```diff
@@ -203,7 +203,6 @@ func (s *Action) InitOnboarding(c *cli.Context) error {
 		return nil
 	}
 
-	ctx = out.AddPrefix(ctx, "[init] ")
```
Comment (AnomalRoil) [inline internal/action/init.go:206]: Not sure why you removed it, since it can be useful in the logs, but I guess we don't really need it neither.

**Review by AnomalRoil — APPROVED** [review]
I guess the one thing we are missing is a test that runs the wizard and create a key and check it works, otherwise we would have caught this a long time ago. 
But IDK how if this can be done easily with our current test setup.

File: internal/action/init.go (hunk 1 above)
Comment (dominikschulz) [reply internal/action/init.go:206]: It was duplicated. In my tests there was still one `[init]` prefix left. Before this change there were two, which felt a bit weird.

**Comment by dominikschulz** [comment]
I won't have time to add tests for this myself anytime soon, but I've filed #1702 to track.

**Pull request merged**